pkg/network/state.go

```diff
@@ -717,15 +740,10 @@ func (ns *networkState) GetStats() map[string]interface{} {
 	return map[string]interface{}{
 		"clients": clientInfo,
 		"telemetry": map[string]int64{
-			"stats_underflows":        ns.telemetry.statsUnderflows,
-			"stats_cookie_collisions": ns.telemetry.statsCookieCollisions,
-			"closed_conn_dropped":     ns.telemetry.closedConnDropped,
-			"conn_dropped":            ns.telemetry.connDropped,
-			"time_sync_collisions":    ns.telemetry.timeSyncCollisions,
-			"dns_stats_dropped":       ns.telemetry.dnsStatsDropped,
-			"http_stats_dropped":      ns.telemetry.httpStatsDropped,
-			"http2_stats_dropped":     ns.telemetry.http2StatsDropped,
-			"dns_pid_collisions":      ns.telemetry.dnsPidCollisions,
+			"closed_conn_dropped": closedConnDropped.Load(),
```

Comment:
Do we still need these to be reported with `GetStats`?

Reply:
yes with the exception of `http2_stats_dropped`, which I will remove. the others are used in the state delta calculation and returned as part of `/connections`